{"nodes":[{"content":"The base exception class for the <xref href=\"System.Data.SqlTypes\"></xref>.","nodes":[{"pos":[0,75],"content":"The base exception class for the <ph id=\"ph1\">&lt;xref href=\"System.Data.SqlTypes\"&gt;&lt;/xref&gt;</ph>.","source":"The base exception class for the <xref href=\"System.Data.SqlTypes\"></xref>."}],"pos":[784,860],"yaml":true},{"content":"Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class.","nodes":[{"pos":[0,99],"content":"Initializes a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Data.SqlTypes.SqlTypeException\"&gt;&lt;/xref&gt;</ph> class.","source":"Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class."}],"pos":[3242,3342],"yaml":true},{"content":"Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class with a specified error message.","nodes":[{"pos":[0,130],"content":"Initializes a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Data.SqlTypes.SqlTypeException\"&gt;&lt;/xref&gt;</ph> class with a specified error message.","source":"Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class with a specified error message."}],"pos":[4827,4958],"yaml":true},{"content":"The error message that explains the reason for the exception.","nodes":[{"pos":[0,61],"content":"The error message that explains the reason for the exception.","nodes":[{"content":"The error message that explains the reason for the exception.","pos":[0,61]}]}],"pos":[5102,5164],"yaml":true},{"content":"Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class with serialized data.","nodes":[{"pos":[0,120],"content":"Initializes a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Data.SqlTypes.SqlTypeException\"&gt;&lt;/xref&gt;</ph> class with serialized data.","source":"Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class with serialized data."}],"pos":[6877,6998],"yaml":true},{"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream. For more information, see [XML and SOAP Serialization](~/docs/framework/serialization/xml-and-soap-serialization.md).","nodes":[{"pos":[0,231],"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream. For more information, see [XML and SOAP Serialization](~/docs/framework/serialization/xml-and-soap-serialization.md).","nodes":[{"content":"This constructor is called during deserialization to reconstitute the exception object transmitted over a stream.","pos":[0,113]},{"content":"For more information, see <bpt id=\"p1\">[</bpt>XML and SOAP Serialization<ept id=\"p1\">](~/docs/framework/serialization/xml-and-soap-serialization.md)</ept>.","pos":[114,231],"source":" For more information, see [XML and SOAP Serialization](~/docs/framework/serialization/xml-and-soap-serialization.md)."}]}],"pos":[7009,7241],"yaml":true,"extradata":"MT"},{"content":"The object that holds the serialized object data.","nodes":[{"pos":[0,49],"content":"The object that holds the serialized object data.","nodes":[{"content":"The object that holds the serialized object data.","pos":[0,49]}]}],"pos":[7501,7551],"yaml":true},{"content":"The contextual information about the source or destination.","nodes":[{"pos":[0,59],"content":"The contextual information about the source or destination.","nodes":[{"content":"The contextual information about the source or destination.","pos":[0,59]}]}],"pos":[7641,7701],"yaml":true},{"content":"Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.","nodes":[{"pos":[0,205],"content":"Initializes a new instance of the <ph id=\"ph1\">&lt;xref href=\"System.Data.SqlTypes.SqlTypeException\"&gt;&lt;/xref&gt;</ph> class with a specified error message and a reference to the inner exception that is the cause of this exception.","source":"Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception."}],"pos":[9136,9342],"yaml":true},{"content":"The message that describes the exception. The caller of this constructor is required to ensure that this string has been localized for the current system culture.","nodes":[{"pos":[0,162],"content":"The message that describes the exception. The caller of this constructor is required to ensure that this string has been localized for the current system culture.","nodes":[{"content":"The message that describes the exception. The caller of this constructor is required to ensure that this string has been localized for the current system culture.","pos":[0,162],"nodes":[{"content":"The message that describes the exception.","pos":[0,41]},{"content":"The caller of this constructor is required to ensure that this string has been localized for the current system culture.","pos":[42,162]}]}]}],"pos":[9499,9662],"yaml":true},{"content":"The exception that is the cause of the current exception. If the <code>innerException</code> parameter is not `null`, the current exception is raised in a `catch` block that handles the inner exception.","nodes":[{"pos":[0,202],"content":"The exception that is the cause of the current exception. If the <code>innerException</code> parameter is not `null`, the current exception is raised in a `catch` block that handles the inner exception.","nodes":[{"content":"The exception that is the cause of the current exception.","pos":[0,57]},{"content":"If the <bpt id=\"p1\">&lt;code&gt;</bpt><ph id=\"ph1\">innerException</ph><ept id=\"p1\">&lt;/code&gt;</ept> parameter is not <ph id=\"ph2\">`null`</ph>, the current exception is raised in a <ph id=\"ph3\">`catch`</ph> block that handles the inner exception.","pos":[58,202],"source":" If the <code>innerException</code> parameter is not `null`, the current exception is raised in a `catch` block that handles the inner exception."}]}],"pos":[9722,9925],"yaml":true}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.Data.SqlTypes.SqlTypeException\n  commentId: T:System.Data.SqlTypes.SqlTypeException\n  id: SqlTypeException\n  children:\n  - System.Data.SqlTypes.SqlTypeException.#ctor\n  - System.Data.SqlTypes.SqlTypeException.#ctor(System.String)\n  - System.Data.SqlTypes.SqlTypeException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Data.SqlTypes.SqlTypeException.#ctor(System.String,System.Exception)\n  langs:\n  - csharp\n  name: SqlTypeException\n  nameWithType: SqlTypeException\n  fullName: System.Data.SqlTypes.SqlTypeException\n  type: Class\n  assemblies:\n  - System.Data.SqlClient\n  - System.Data\n  - netstandard\n  - System.Data.Common\n  namespace: System.Data.SqlTypes\n  summary: The base exception class for the <xref href=\"System.Data.SqlTypes\"></xref>.\n  syntax:\n    content: 'public class SqlTypeException : SystemException'\n  inheritance:\n  - System.Object\n  - System.Exception\n  derivedClasses:\n  - System.Data.SqlTypes.SqlAlreadyFilledException\n  - System.Data.SqlTypes.SqlNotFilledException\n  - System.Data.SqlTypes.SqlNullValueException\n  - System.Data.SqlTypes.SqlTruncateException\n  implements: []\n  inheritedMembers:\n  - System.Exception.Data\n  - System.Exception.GetBaseException\n  - System.Exception.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Exception.GetType\n  - System.Exception.HelpLink\n  - System.Exception.HResult\n  - System.Exception.InnerException\n  - System.Exception.Message\n  - System.Exception.SerializeObjectState\n  - System.Exception.Source\n  - System.Exception.StackTrace\n  - System.Exception.TargetSite\n  - System.Exception.ToString\n  - System.Object.Equals(System.Object)\n  - System.Object.Equals(System.Object,System.Object)\n  - System.Object.GetHashCode\n  - System.Object.MemberwiseClone\n  - System.Object.ReferenceEquals(System.Object,System.Object)\n  version:\n  - netcore-1.0\n  - netcore-1.1\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  monikers:\n  - netcore-1.0\n  - netcore-1.1\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.SqlTypes/SqlTypeException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\n- uid: System.Data.SqlTypes.SqlTypeException.#ctor\n  commentId: M:System.Data.SqlTypes.SqlTypeException.#ctor\n  id: '#ctor'\n  parent: System.Data.SqlTypes.SqlTypeException\n  langs:\n  - csharp\n  name: SqlTypeException()\n  nameWithType: SqlTypeException.SqlTypeException()\n  fullName: SqlTypeException.SqlTypeException()\n  type: Constructor\n  assemblies:\n  - System.Data.SqlClient\n  - System.Data\n  - netstandard\n  - System.Data.Common\n  namespace: System.Data.SqlTypes\n  summary: Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class.\n  syntax:\n    content: public SqlTypeException ();\n    parameters: []\n  overload: System.Data.SqlTypes.SqlTypeException.#ctor*\n  exceptions: []\n  version:\n  - netcore-1.0\n  - netcore-1.1\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  monikers:\n  - netcore-1.0\n  - netcore-1.1\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.SqlTypes/SqlTypeException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\n- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.String)\n  commentId: M:System.Data.SqlTypes.SqlTypeException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.Data.SqlTypes.SqlTypeException\n  langs:\n  - csharp\n  name: SqlTypeException(String)\n  nameWithType: SqlTypeException.SqlTypeException(String)\n  fullName: SqlTypeException.SqlTypeException(String)\n  type: Constructor\n  assemblies:\n  - System.Data.SqlClient\n  - System.Data\n  - netstandard\n  - System.Data.Common\n  namespace: System.Data.SqlTypes\n  summary: Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class with a specified error message.\n  syntax:\n    content: public SqlTypeException (string message);\n    parameters:\n    - id: message\n      type: System.String\n      description: The error message that explains the reason for the exception.\n  overload: System.Data.SqlTypes.SqlTypeException.#ctor*\n  exceptions: []\n  version:\n  - netcore-1.0\n  - netcore-1.1\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  monikers:\n  - netcore-1.0\n  - netcore-1.1\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.SqlTypes/SqlTypeException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\n- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  commentId: M:System.Data.SqlTypes.SqlTypeException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.Data.SqlTypes.SqlTypeException\n  langs:\n  - csharp\n  name: SqlTypeException(SerializationInfo, StreamingContext)\n  nameWithType: SqlTypeException.SqlTypeException(SerializationInfo, StreamingContext)\n  fullName: SqlTypeException.SqlTypeException(SerializationInfo, StreamingContext)\n  type: Constructor\n  assemblies:\n  - System.Data\n  - netstandard\n  - System.Data.Common\n  namespace: System.Data.SqlTypes\n  summary: Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class with serialized data.\n  remarks: This constructor is called during deserialization to reconstitute the exception object transmitted over a stream. For more information, see [XML and SOAP Serialization](~/docs/framework/serialization/xml-and-soap-serialization.md).\n  syntax:\n    content: protected SqlTypeException (System.Runtime.Serialization.SerializationInfo si, System.Runtime.Serialization.StreamingContext sc);\n    parameters:\n    - id: si\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The object that holds the serialized object data.\n    - id: sc\n      type: System.Runtime.Serialization.StreamingContext\n      description: The contextual information about the source or destination.\n  overload: System.Data.SqlTypes.SqlTypeException.#ctor*\n  exceptions: []\n  version:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  monikers:\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.SqlTypes/SqlTypeException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\n- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.String,System.Exception)\n  commentId: M:System.Data.SqlTypes.SqlTypeException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.Data.SqlTypes.SqlTypeException\n  langs:\n  - csharp\n  name: SqlTypeException(String, Exception)\n  nameWithType: SqlTypeException.SqlTypeException(String, Exception)\n  fullName: SqlTypeException.SqlTypeException(String, Exception)\n  type: Constructor\n  assemblies:\n  - System.Data.SqlClient\n  - System.Data\n  - netstandard\n  - System.Data.Common\n  namespace: System.Data.SqlTypes\n  summary: Initializes a new instance of the <xref href=\"System.Data.SqlTypes.SqlTypeException\"></xref> class with a specified error message and a reference to the inner exception that is the cause of this exception.\n  syntax:\n    content: public SqlTypeException (string message, Exception e);\n    parameters:\n    - id: message\n      type: System.String\n      description: The message that describes the exception. The caller of this constructor is required to ensure that this string has been localized for the current system culture.\n    - id: e\n      type: System.Exception\n      description: The exception that is the cause of the current exception. If the <code>innerException</code> parameter is not `null`, the current exception is raised in a `catch` block that handles the inner exception.\n  overload: System.Data.SqlTypes.SqlTypeException.#ctor*\n  exceptions: []\n  version:\n  - netcore-1.0\n  - netcore-1.1\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  monikers:\n  - netcore-1.0\n  - netcore-1.1\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.SqlTypes/SqlTypeException.xml\n  ms.technology:\n  - dotnet-ado\n  author: JennieHubbard\n  ms.author: jhubbard\n  manager: jhubbard\nreferences:\n- uid: System.Exception\n  parent: System\n  isExternal: false\n  name: Exception\n  nameWithType: Exception\n  fullName: System.Exception\n- uid: System.Data.SqlTypes.SqlTypeException.#ctor\n  parent: System.Data.SqlTypes.SqlTypeException\n  isExternal: false\n  name: SqlTypeException()\n  nameWithType: SqlTypeException.SqlTypeException()\n  fullName: SqlTypeException.SqlTypeException()\n- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.String)\n  parent: System.Data.SqlTypes.SqlTypeException\n  isExternal: false\n  name: SqlTypeException(String)\n  nameWithType: SqlTypeException.SqlTypeException(String)\n  fullName: SqlTypeException.SqlTypeException(String)\n- uid: System.String\n  parent: System\n  isExternal: false\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Data.SqlTypes.SqlTypeException\n  isExternal: false\n  name: SqlTypeException(SerializationInfo, StreamingContext)\n  nameWithType: SqlTypeException.SqlTypeException(SerializationInfo, StreamingContext)\n  fullName: SqlTypeException.SqlTypeException(SerializationInfo, StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.Data.SqlTypes.SqlTypeException.#ctor(System.String,System.Exception)\n  parent: System.Data.SqlTypes.SqlTypeException\n  isExternal: false\n  name: SqlTypeException(String, Exception)\n  nameWithType: SqlTypeException.SqlTypeException(String, Exception)\n  fullName: SqlTypeException.SqlTypeException(String, Exception)\n- uid: System.Data.SqlTypes.SqlTypeException.#ctor*\n  parent: System.Data.SqlTypes.SqlTypeException\n  isExternal: false\n  name: SqlTypeException\n  nameWithType: SqlTypeException.SqlTypeException\n  fullName: SqlTypeException.SqlTypeException\n  monikers:\n  - netcore-1.0\n  - netcore-1.1\n  - netcore-2.0\n  - netframework-4.5.1\n  - netframework-4.5.2\n  - netframework-4.5\n  - netframework-4.6.1\n  - netframework-4.6.2\n  - netframework-4.6\n  - netframework-4.7\n  - netstandard-2.0\n  - xamarinandroid-7.1\n  - xamarinios-10.8\n  - xamarinmac-3.0\n  content_git_url: https://github.com/dotnet/docs/blob/master/xml/System.Data.SqlTypes/SqlTypeException.xml\n- uid: System.Object.Equals(System.Object)\n  parent: System.Object\n  isExternal: false\n  name: Equals(Object)\n  nameWithType: Object.Equals(Object)\n  fullName: Object.Equals(Object)\n- uid: System.Object.Equals(System.Object,System.Object)\n  parent: System.Object\n  isExternal: false\n  name: Equals(Object, Object)\n  nameWithType: Object.Equals(Object, Object)\n  fullName: Object.Equals(Object, Object)\n- uid: System.Object.GetHashCode\n  parent: System.Object\n  isExternal: false\n  name: GetHashCode()\n  nameWithType: Object.GetHashCode()\n  fullName: Object.GetHashCode()\n- uid: System.Exception.GetType\n  parent: System.Exception\n  isExternal: false\n  name: GetType()\n  nameWithType: Exception.GetType()\n  fullName: Exception.GetType()\n- uid: System.Object.MemberwiseClone\n  parent: System.Object\n  isExternal: false\n  name: MemberwiseClone()\n  nameWithType: Object.MemberwiseClone()\n  fullName: Object.MemberwiseClone()\n- uid: System.Object.ReferenceEquals(System.Object,System.Object)\n  parent: System.Object\n  isExternal: false\n  name: ReferenceEquals(Object, Object)\n  nameWithType: Object.ReferenceEquals(Object, Object)\n  fullName: Object.ReferenceEquals(Object, Object)\n- uid: System.Exception.ToString\n  parent: System.Exception\n  isExternal: false\n  name: ToString()\n  nameWithType: Exception.ToString()\n  fullName: Exception.ToString()\n- uid: System.Exception.Data\n  parent: System.Exception\n  isExternal: false\n  name: Data\n  nameWithType: Exception.Data\n  fullName: Exception.Data\n- uid: System.Exception.GetBaseException\n  parent: System.Exception\n  isExternal: false\n  name: GetBaseException()\n  nameWithType: Exception.GetBaseException()\n  fullName: Exception.GetBaseException()\n- uid: System.Exception.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Exception\n  isExternal: false\n  name: GetObjectData(SerializationInfo, StreamingContext)\n  nameWithType: Exception.GetObjectData(SerializationInfo, StreamingContext)\n  fullName: Exception.GetObjectData(SerializationInfo, StreamingContext)\n- uid: System.Exception.HelpLink\n  parent: System.Exception\n  isExternal: false\n  name: HelpLink\n  nameWithType: Exception.HelpLink\n  fullName: Exception.HelpLink\n- uid: System.Exception.HResult\n  parent: System.Exception\n  isExternal: false\n  name: HResult\n  nameWithType: Exception.HResult\n  fullName: Exception.HResult\n- uid: System.Exception.InnerException\n  parent: System.Exception\n  isExternal: false\n  name: InnerException\n  nameWithType: Exception.InnerException\n  fullName: Exception.InnerException\n- uid: System.Exception.Message\n  parent: System.Exception\n  isExternal: false\n  name: Message\n  nameWithType: Exception.Message\n  fullName: Exception.Message\n- uid: System.Exception.SerializeObjectState\n  parent: System.Exception\n  isExternal: false\n  name: SerializeObjectState\n  nameWithType: Exception.SerializeObjectState\n  fullName: Exception.SerializeObjectState\n- uid: System.Exception.Source\n  parent: System.Exception\n  isExternal: false\n  name: Source\n  nameWithType: Exception.Source\n  fullName: Exception.Source\n- uid: System.Exception.StackTrace\n  parent: System.Exception\n  isExternal: false\n  name: StackTrace\n  nameWithType: Exception.StackTrace\n  fullName: Exception.StackTrace\n- uid: System.Exception.TargetSite\n  parent: System.Exception\n  isExternal: false\n  name: TargetSite\n  nameWithType: Exception.TargetSite\n  fullName: Exception.TargetSite\n"}